CorsixTH/Lua/languages/czech.lua

```diff
@@ -3029,117 +3091,117 @@ staff_list = {
-    "Pokud nezabijete příliš mnoho lidí a udržíte je rozumně šťastné, neměli byste mít v této úrovni problémy!//" ..
+    "Malé varování - opravdu si hlídejte Vaši reputaci - to je to, co pacienty přiláká z široka daleka do vašeho zařízení. " ..
+    "Pokud nezabijete příliš mnoho lidí a udržíte je rozumně šťastné, neměli byste mít v této úrovni problémy!\n\n" ..
     "Teď jste v tom sami. Hodně štěstí a tak dále.",
   level4 =
-    "Udržte všechny lidi šťastné, zacházejte s nimi tak dobře jak to jen a udržte úmrtí na absolutní minimum." ..
-    "Vaše reputace je v sázce, takže se ujistěte, že ji budete mít tak vysoko, jak je to jen možné. " ..
-    "O peníze se moc nestarejte - ty přijdou, jak Vaše důležitá reputace poroste. " ..
-    "Budete mít také možnost učit Doktory k rozšíření jejich schopností. " ..
-    "Mohli by mít na starost pacienta, který se zdá být méně neprůhledný, než většina. " ..
+    "Udržte všechny lidi šťastné, zacházejte s nimi tak dobře, jak to jen jde, a udržte úmrtí na absolutním minimu. " ..
+    "V sázce je vaše reputace, takže se ujistěte, že ji budete mít tak vysoko, jak je to jen možné. " ..
+    "O peníze se moc nestarejte - ty přijdou společně s růstem vaší reputace. " ..
+    "Budete mít také možnost učit Doktory a rozšířit jejich schopnosti. " ..
+    "Mohli by mít na starost pacienta, který se zdá být méně neprůhledný než ostatní. " ..
     "Dosáhněte reputace přes 500.",
   level13 =
-    "Vaše neuvěřitelné dovednosti správce nemocnice neunikly Zvláštnímu Tajnému Oddělení Zvláštní Tajné. " ..
-    "Mají pro Vás zvláštní bonus; je tu krysami zamořená nemocnice, která potřebuje účinného Terminátora. " ..
+    "Vaše neuvěřitelné dovednosti správce nemocnice neunikly Bezpečnostní Informační Službě. " ..
+    "Mají pro vás zvláštní bonus; je tu krysami zamořená nemocnice, která potřebuje účinného Terminátora. " ..
     "Musíte zastřelit tolik krys, kolik můžete, než Údržbáři uklidí všechen bordel. " ..
     "Myslíte, že to zvládnete?",
   level6 =
-    "Použijte všechny Vaše znalosti k zřízení bezproblémové, dobře spravované nemocnice, která má zdravý příjem a může se vypořádat s čímkoliv, co na ní nemocná veřejnost může hodit. " ..
+    "Použijte všechny vaše znalosti k zřízení bezproblémové, dobře spravované nemocnice, která má zdravý příjem a může se vypořádat s čímkoliv, co na ni nemocná veřejnost může hodit. " ..
     "Musíte si uvědomit, že zdejší atmosféra je zvláště vhodná k přenášení bacilů a infekcí. " ..
-    "Pokud nebudete Vaše zařízení udržovat až puntičkářsky čisté, můžete čelit řadě epidemií mezi pacienty. " ..
-    "Ujistěte se, že si vyděláte 150,000€ a že Vaše nemocnice má hodnotu přes 140,000€.",
+    "Pokud nebudete vaše zařízení udržovat až puntičkářsky čisté, můžete čelit řadě epidemií mezi pacienty. " ..
+    "Ujistěte se, že si vyděláte 150,000€ a že vaše nemocnice má hodnotu přes 140,000€.",
```

Comment:
The translation file has some uses of €, and some of $.
While ideally we should support other currencies in our UI, the game does kind of specifically want to push us to $ in its current state.